core/services/workflows/syncer/handler_test.go

```diff
@@ -321,12 +321,12 @@ func Test_workflowDeletedHandler(t *testing.T) {
 			})
 		)
 
-		giveWFID := workflowID(binary, config, []byte(secretsURL))
+		giveWFID, err := pkgworkflows.GenerateWorkflowID(wfOwner, binary, config, secretsURL)
 
-		b, err := hex.DecodeString(giveWFID)
 		require.NoError(t, err)
 		wfID := make([]byte, 32)
-		copy(wfID, b)
+		copy(wfID, giveWFID[:])
+		wfIDs := hex.EncodeToString(wfID[:])
```

Comment:
The linter flagged an unnecessary slice of `wfID`, but could `wfID` be dropped entirely? I see one usage below, but it is casting to the same type, and it looks like it could just slice `giveWFID` directly.

Reply:
Yeah good point -- I mechanically refactored this and didn't think much of it, but have addressed it now :)